Update monitor configuration to reflect evaluation frequency (#14161)
DataDog/documentation: content/en/monitors/create/configuration.md

**Note**: There are different behaviors when utilizing `as_count()`. See [as_count() in Monitor Evaluations][1] for details.

### Evaluation window

Monitors are evaluated at a certain frequency looking back at the last `5 minutes`, `15 minutes`, `1 hour`, etc.

### Evaluation frequency

> The evaluation frequency is how often the monitor query is being performed by Datadog. For most configurations, the evaluation frequency is `1 minute`, which means that every minute, the monitor queries the [selected data](#define-the-search-query) over the [selected evaluation window](#evaluation-window) and compares the aggregated value against the [defined thresholds](#thresholds).

Reviewer: ```suggestion
The evaluation frequency defines how often Datadog performs the monitor query. For most configurations, the evaluation frequency is `1 minute`, which means that every minute, the monitor queries the [selected data](#define-the-search-query) over the [selected evaluation window](#evaluation-window) and compares the aggregated value against the [defined thresholds](#thresholds).
```
